test/metabase/api/card_test.clj

```diff
@@ -11,17 +11,28 @@
             [metabase.api.pivots :as pivots]
             [metabase.driver.sql-jdbc.execute :as sql-jdbc.execute]
             [metabase.http-client :as http]
-            [metabase.models :refer [Card CardFavorite Collection Dashboard Database ModerationReview
-                                     Pulse PulseCard PulseChannel PulseChannelRecipient Table ViewLog]]
+            [metabase.models
```

Comment:
This change is probably not desired. Auto format got a bit hungry

Reply:
good catch. I'll fix